src/Confluent.Kafka/Library.cs

```diff
@@ -57,5 +57,39 @@ public static string VersionString
         public static string[] DebugContexts
             => Util.Marshal.PtrToStringUTF8(LibRdKafka.get_debug_contexts()).Split(',');
 
+        /// <summary>
+        ///     true if librdkafka has been successfully loaded, false if not.
+        /// </summary>
+        public static bool IsLoaded
+            => LibRdKafka.IsInitialized;
+
+        /// <summary>
+        ///     Loads the native librdkafka library. Does nothing if the library is
+        ///     already loaded.
+        /// </summary>
+        /// <returns>
+        ///     true if librdkafka was loaded as a result of this call, false otherwise.
```

Comment:
Will loading exceptions be raised here?

Reply:
i will clarify. it should read 'false if the library has already been loaded prior to this call'